src/primitives/logpdf.jl

```diff
@@ -17,6 +17,8 @@ end
         type2model(_m) |> sourceLogpdf() |> loadvals(_args, _data))
 end
 
+sourceLogpdf(m::Model) = sourceLogpdf()(m)
+
```

Comment:
Do we want to allow this to work on joint distributions too?
Also, it is intended to not be exported?

Reply:
I wasn't sure about exporting these. Maybe we should? I guess those names are very unlikely to collide.

These are mostly for interactive use, but I guess it makes sense to define them for joint distributions too

Reply:
I had noticed that `sourceRand` works for JointDistributions and is exported, so it might be confusing if this isn't.

I think either way is fine. I can't find a good way to list all of the explicitly exported functions in a module, so we'd ultimately just want to document them and it would matter less

Reply:
Changed this so they're exported